pkg/restart/restart_windows.go

```diff
@@ -2,10 +2,13 @@
 
 package restart
 
-import "os/exec"
+import (
+	"os/exec"
+)
 
-func attachTTY(*exec.Cmd) {
+func attachTTY(_ *exec.Cmd) {
 	// not used for Windows
+}
 
-	return
+func (c *Context) ignoreSignals() {
 }
```

Comment:
What are these blank functions?

Reply:
placeholder function, to match the unix equivalent